packages/gatsby-source-contentful/src/extend-node-type.js

```diff
@@ -594,13 +649,30 @@ exports.extendNodeType = ({ type, store, cache, getNodesByType }) => {
 
   const fluidNode = fluidNodeType({ name: `ContentfulFluid`, getTracedSVG })
   const sizesNode = fluidNodeType({ name: `ContentfulSizes`, getTracedSVG })
+  const gatsbyImageData = getGatsbyImageFieldConfig(resolveGatsbyImageData, {
+    jpegProgressive: {
+      type: GraphQLBoolean,
+      defaultValue: true,
+    },
+    resizingBehavior: {
+      type: ImageResizingBehavior,
+    },
+    cropFocus: {
+      type: ImageCropFocusType,
+    },
+    quality: {
+      type: GraphQLInt,
+      defaultValue: 50,
+    },
+  })
   sizesNode.deprecationReason = `Sizes was deprecated in Gatsby v2. It's been renamed to "fluid" https://example.com/write-docs-and-fix-this-example-link`
```

Comment:
Should we deprecate all old resolvers? Aka fluid and sizes?

Reply:
We're not going to deprecate them in transformer-sharp yet. We'll probably add a deprecation message in a major early next year.

Reply:
Alright 🆗